CHANGELOG.md

```diff
@@ -58,17 +58,17 @@ VPP v18.04-rc0~90-gd95c39e
 cn-infra v1.1
 
 ### Improvements
-- [aclplugin](plugins/defaultplugins/aclplugin) 
+- [aclplugin](plugins/vpp/aclplugin) 
   * Improved resync of ACL entries. Every new ACL entry is correctly configured in the VPP and all obosolete entries are read and removed. 
-- [ifplugin](plugins/defaultplugins/ifplugin) 
+- [ifplugin](plugins/vpp/ifplugin) 
   * Improved resync of interfaces, BFD sessions, authentication keys, echo functions and STN. Better resolution of persistence config for interfaces. 
-- [l2plugin](plugins/defaultplugins/l2plugin) 
+- [l2plugin](plugins/vpp/l2plugin) 
   * Improved resync of bridge domains, FIB entries and xConnect pairs. Resync now better correlates configuration present on the VPP with the NB setup.
-- (Linux) [ifplugin](plugins/linuxplugin/l3plugin) 
+- (Linux) [ifplugin](plugins/linux/l3plugin) 
   * ARP does not need the interface to be present on the VPP. Configuration is cached and put to the VPP if requirements are fullfiled. 
 
 ### Fixes
-  * [vpp-agent-grpc](cmd/vpp-agent-grpc) now compiles properly
+  * [vpp-agent-grpc](cmd/vpp-agent) (removed in 1.4 release) now compiles properly
```

Comment:
I would suggest mentioning that it was merged into **vpp-agent** executable and not removed, not confuse someone that we dropped grpc support.